Config/BuildSettings.swift

```diff
@@ -0,0 +1,33 @@
+// 
+// Copyright 2020 Vector Creations Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+/// BuildSettings provides settings computed at build time.
+/// In future, it may be automatically generated from xcconfig files
+@objcMembers
+final class BuildSettings: NSObject {
+    
+    static var bundleDisplayName: String {
+        Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as! String
```

Comment:
```suggestion
        Bundle.app.object(forInfoDictionaryKey: "CFBundleDisplayName") as! String
```

Reply:
We cannot use `Bundle.main` in extensions, we can use an extension like above to get real app bundle, or we'll need to include keys also in _Info.plist_'s of the extensions.